globus_harvester.py

```diff
@@ -1,14 +1,15 @@
 """Globus Harvester.
 
 Usage:
-  globus_harvester.py [--onlyharvest | --onlyexport] [--only-new-records] [--export-filepath=<file>] [--export-format=<format>]
+  globus_harvester.py [--onlyharvest | --onlyexport] [--only-new-records] [--export-filepath=<file>] [--export-format=<format>] [--init]
```

Comment:
looks good but I think `--init` should be mutually exclusive with `--onlyharvest` and `--onlyexport`, which it isn't here.